wiki/Tournaments/3WC/2021/en.md

```diff
@@ -0,0 +1,122 @@
+| Designer | ![][flag_US] [Celsea](https://osu.ppy.sh/u/7464885) |
+| Statistician | ![][flag_FI] [shdewz](https://osu.ppy.sh/u/10000899) |
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1256770)
+- **[Main Spreadsheet](https://docs.google.com/spreadsheets/d/1oLTe6_WStAfmNU2b6RhyAvZXKkp23dhp5LM6vL6cYuM "Google Spreadsheet")**
+- [Discord server](https://discord.gg/ZjPs7ya)
+- [Challonge brackets](https://challonge.com/3WC2021)
+- [Pick'ems page](https://pickem.hwc.hr/tournaments/54) hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)
+- [Livestream](https://www.twitch.tv/3wc_osu "Twitch")
+
+## Ruleset
+
+### Tournament rules
+
+1. The 3 digit world cup is a world cup style tournament aimed for players in the 3 digit rank range (#100-#999) of osu!standard without the enforcement of a Badge Weighting System (BWS).
+2. Team size is 4-6 players, and matches are played in a 3v3 format.
+3. Beatmap scoring is based on Score V2.
+4. After signups close, players that didn't get a minimum roster for their country will be able to form a Countryless team, with a maximum of 1 player per country. 
+5. Qualifiers will be used to seed the teams into a 16 team double elimination bracket.
+    - The qualifiers mappool will be 10 maps (4NM/2HD/2HR/2DT) and will be played twice. The best combined team run will be used for seeding, and teams are allowed to skip maps in the second run where they believe they already have a good run on.
+6. Grand Finals will have a bracket reset if the team coming from the losers bracket wins the first match.
+7. Match schedules and mappools will be released on the Monday before those matches should be played, and schedules will be written in UTC+0.
+8. As per standard tournament guidelines, staff other than streamers and commentators are not allowed to play.
+
+### Tournament registrations
+
+1. Each player will sign up individually
+    - Players must be in the rank range by the registrations closing. 
+2. A captain will be designated from the players that signed up and will be responsible for the well-being of the team.
+    - The assigned captain may give his role to another player, and even exclude themselves from the team if they believe that is the right move.
+    - If a country has more signups than the team size allows for, the captain will be tasked with forming the team.
+
+### Match instructions
+
+- Round of 16 and Quarterfinals
+    - Best of 11, 2 Bans, 5NM/2HD/2HR/3DT/2FM/1TB
+- Semifinals
+    - Best of 11, 2 Bans, 6NM/3HD/3HR/4DT/3FM/1TB
+- Finals and Grand Finals
+    - Best of 13, 2 Bans, 6NM/3HD/3HR/4DT/3FM/1TB
+
+1. The referee will invite the captains of each team to the multiplayer lobby. Captains have 10 minutes to get a minimum of a 3 man roster in the lobby, otherwise the opposing team will be allowed to claim a win by default. 
```

Comment:
```suggestion
1. The referee will invite the captains of each team to the multiplayer lobby. Captains have ten minutes to get a minimum of a three-man roster in the lobby, otherwise the opposing team will be allowed to claim a win by default. 
```